server/integration/testsuite/src/test/java/org/infinispan/server/test/cs/jdbc/SingleNodeJdbcStoreIT.java

```diff
@@ -414,9 +422,9 @@ public boolean isSatisfied() throws Exception {
     public String getStoredKey(RemoteCache cache, String key) throws IOException, InterruptedException {
         // 1. marshall the key
         // 2. encode it with base64 (that's what DefaultTwoWayKey2StringMapper does)
-        // 3. prefix it with 9 (again, done by DefaultTwoWayKey2StringMapper to mark the key as byte array type)
+        // 3. prefix it with 8 (again, done by DefaultTwoWayKey2StringMapper to mark the key as wrapped byte array type)
         // 4. prefix it with UTF-16 BOM (that is what DefaultTwoWayKey2StringMapper does for non string values)
-        return '\uFEFF' + "9" + Base64.encodeBytes(cache.getRemoteCacheManager().getMarshaller().objectToByteBuffer(key));
+        return '\uFEFF' + "8" + Base64.encodeBytes(cache.getRemoteCacheManager().getMarshaller().objectToByteBuffer(key));
```

Comment:
I was thinking about `storeAsBinary` in 9.0.0.Alpha1 vs 9.0.0.Beta1. I suspect the values do change, because we're including the byte array length with the encoding scheme used by `MarshallUtil.marshallInt`, whereas Alpha1 is probably just passing the byte array to the store.